src/komal/coding_exam_1/DiffBetweenYoungerAndanElderMember.java

```diff
@@ -0,0 +1,35 @@
+package komal.coding_exam_1;
+/*WAP to provide the difference between a Younger and an elder member of the family. Age is provided in the form of an array.
+
+Sample Input: 7 12 56 62 25
+
+Sample Output: 55*/
+
+public class DiffBetweenYoungerAndanElderMember {
+
+	int getYoungestMemberAge(int[] age) {
+		int minAge = age[0];
```

Comment:
This can be handle with single loop with max and min number, not required two different methods